tests/kj-rs/lib.rs

```diff
@@ -52,4 +58,32 @@ mod tests {
         let _ = ffi::c_async_int_fn();
         let _ = ffi::c_async_struct_fn();
     }
+
+    #[test]
+    fn kj_own() {
+        let mut own = ffi::cpp_kj_own();
+        // Methods on C++ classes can be called from Rust
+        assert_eq!(own.cpptype_get(), 42);
+        own.pin_mut().cpptype_set(14);
+        assert_eq!(own.cpptype_get(), 14);
+        // Explicitly drop for clarity / debugging drop impl
+        std::mem::drop(own);
+    }
+
+    #[test]
+    fn kj_move() {
+        let owned = ffi::cpp_kj_own();
+        // Move owned into moved_value
+        let mut moved_value = owned;
+        assert_eq!(moved_value.cpptype_get(), 42);
+        moved_value.pin_mut().cpptype_set(14);
```

Comment:
to make this more fun, call set on `owned` before move and verify that it is the same instance?